app/_kong_plugins/acme/index.md

```diff
@@ -29,10 +29,217 @@ icon: acme.png
 
 categories:
   - security
+  - certificates
 
 search_aliases:
   - let's encrypt
   - certificates
 ---
 
-## Overview
+The ACME plugin allows {{site.base_gateway}} to apply certificates from Let's Encrypt or any other ACMEv2 service and serve them dynamically. 
+You can also configure a threshold time for automatic renewal. 
+
+## How it works
+
+The ACME plugin uses the [HTTP-01 challenge](https://letsencrypt.org/docs/challenge-types/), which lets you verify domain ownership before issuing an SSL certificate. 
+The plugin generates a challenge token and key thumbprint, then presents them to Let's Encrypt or another ACMEv2 service when requested.
+
+To use this plugin, you need:
+* A public IP and a resolvable DNS
+* This challenge can only be done on port 80, so {{site.base_gateway}} needs to accept proxy traffic this port
```

Comment:
```suggestion
* This challenge can only be done on port 80, so {{site.base_gateway}} needs to accept proxy traffic this port. You can configure this with `proxy_listen` in kong.conf.
```
I think this is correct, at least I learned it from https://docs.konghq.com/gateway/3.9.x/reference/configuration/#proxy_listen